test/Microsoft.AspNetCore.Hosting.Tests/Internal/HostingEventSourceTests.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics.Tracing;
+using System.Reflection;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Hosting.Internal
+{
+    public class HostingEventSourceTests
+    {
+        [Fact]
+        public void MatchesNameAndGuid()
```

Comment:
Is this the only test worth writing? Any way we can wire up an event listener to make sure the events with the right data gets fired?